cmd/copy.go

```diff
@@ -123,6 +123,8 @@ type rawCopyCmdArgs struct {
 	preserveSMBPermissions bool
 	preservePermissions    bool // Separate flag so that we don't get funkiness with two "flags" targeting the same boolean
 	preserveOwner          bool // works in conjunction with preserveSmbPermissions
+	// Default true; false indicates that the destination is the target directory, rather than something we'd put a directory under (e.g. a container)
+	asSubdir bool
```

Comment:
could you please clarify the difference compared to the existing flag `internalOverrideStripTopDir bool` with a comment?